package.json

```diff
@@ -104,7 +105,6 @@
     "@babel/traverse": "^7.23.2",
     "loader-utils": "2.0.4",
     "minimatch": "3.1.2",
-    "json5": "2.2.2",
```

Comment:
Removing it from our `resolutions` since we're now using as a dependency

Reply:
I might be reading the lockfile wrong but it looks like something is still requiring 1.0.2 so if I think removing this resolution would mean both versions are installed.

Reply:
Thanks Jacob!! You're right. I think i have to add it in both the resolutions and in the devdependency?